### ISSUE TITLE
As a site visitor, I want to browse stories by tags via the front page of the site

### ISSUE THREAD

**Issue by paulproteus** [openhatch/oh-contribution-stories#8]
Implementation: define some tags/tag structure, implement via Pelican tag and/or categories system

**Comment by paulproteus** [comment]
```
[19:50:30] <paulproteus> Other fun fact:
[19:50:30] <paulproteus> All templates will receive the variables defined in your settings file, if they are in all-caps. You can access them directly.
[19:50:52] <paulproteus> Therefore, if we define a dict in the settings file that goes from tagslug => tagnicename, then we can use that from our template.
[19:53:03] <paulproteus> sorted([('x', ['x','y','z']), ('y', ['3', '4', '5'])])
[19:58:32] <paulproteus> TAG_TO_NICE_NAME = {'helpedby-irc': "Helped by IRC"}
```

**Comment by shaunagm** [comment]
Closed by 47a0ca6843416998950c10b52a325531d89770c6
